### PR TITLE
Sylvia/docs/doc 1302

### PR DESCRIPTION
**Pull request by srpearce**
@mhoeber What we just talked about: Moved the old mobile FAQ and associated images to SFD, then created a link on the old page pointing to the new one.

### CODE REVIEW

**Comment by mhoeber** [comment]
:+1:

**Comment by srpearce** [comment]
@lamagnifica FYI, Mark and I decided to add the updated mobile FAQ to the student guide, even though Patrick has also added it to edx.org. For future updates, the current plan is to update the student-facing docs here and then send the updated info to Marketing (basically the same thing as we did this time). This process isn't ideal, obviously, and we can discuss it further the next time we need to update the docs.

File: en_us/students/source/change_log.rst [hunk 1]
```diff
@@ -10,6 +10,8 @@ Change Log
 
    * - Date
      - Change
+   * - 12/8/14
```
Comment (lamagnifica) [inline en_us/students/source/change_log.rst]: date?

**Comment by lamagnifica** [comment]
update the change log then it's good to go :+1:

**Comment by srpearce** [comment]
Yikes - thanks for the date reminder. :-)

**Pull request merged**